doc-content/enterprise-only/bpmn/bpmn-user-task-lifecycle-ref.adoc

```diff
@@ -0,0 +1,26 @@
+[id='bpmn-user-task-lifecycle-ref']
+= BPMN2 user task life cycle in process designer
+
+You can trigger a user task element during the process instance execution to create a user task. The user task service of the task execution engine executes the user task instance. The process instance continues the execution only when the associated user task is completed or aborted. A user task life cycle is as follows:
+
+* When a process instance enters a user task element, the user task is in the `Created` stage.
+* `Created` stage is a transient stage and the user task enters the `Ready` stage immediately. The task appears in the task list of all the actors who are allowed to execute the task.
+* When an actor claims the user task, the task becomes `Reserved`.
+
+[NOTE]
+====
+If a user task has a single potential actor, the task is assigned to that actor upon creation.
+====
+
+* When an actor who claimed the user task starts the execution, the status of the user task changes to `InProgress`.
+* Once an actor completes the user task, the status changes to `Completed` or `Failed` depending on the execution outcome.
+
+There are also several other life cycle methods, including:
+
+* Delegating or forwarding a user task so the user task is assigned to another actor.
+* Revoking a user task, then the user task is no longer claimed by a single actor but is available to all actors who are allowed to take it.
+* Suspending and resuming a user task.
+* Stopping a task that is in progress.
```

Comment:
May be here also you can add "**user task**" just to follow the consistency. 
Suggestion: Stopping a **user task** that is in progress.

Correct me if I am wrong.

Reply:
Done, thanks